src/components/navbar/Navbar.component.scss

```
@@ -0,0 +1,22 @@
+@import "src/assets/styles/_variables.scss";
+
+.navbar {
+  background-color: $color-white;
+  color: $primary-color;
+  padding: 0.313rem 12.5rem;
+  display: flex;
+  justify-content: space-between;
+  height: 3.125rem;
```

```suggestion
  background-color: $color-white;
  color: $primary-color;
  display: flex;
  height: 3.125rem;
  justify-content: space-between;
  padding: 0.313rem 12.5rem;
```
Let's follow a-z order for css properties. Please do changes on other classes (where necessary) also. Thanks